liveRecorder/serviceWrappers/linux/liveRecorder

```diff
@@ -44,30 +51,41 @@ createFolders() {
 }
 
 start() {
+     modules=(schedule m3u8 poster psutil pycrypto)
+     for module in "${modules[@]}"; do
+         python -c "import $module"
+         [ $? -ne 0 ] && pip install $module
+    done
 	createFolders || exit 1
-    if [ -f $PIDFILE ]; then
+    status -p ${PIDFILE}
+    if [ $? -eq 0 ]; then
         echo "${NAME} already running"
     else
         echo "Starting ${NAME}"
-        python2.7 $APPLICATION_PATH >> $LOGFILE  2>&1 &
-	    RETVAL=$?
-        echo $! > $PIDFILE
-        if [ $RETVAL -eq 0 ] ; then 
-			echo "started ${NAME} with pid $(cat $PIDFILE)"
+        python2.7 ${APPLICATION_PATH} >> ${LOGFILE}  2>&1 &
+        RETVAL=$?
+        echo $! > ${PIDFILE}
+        if [ ${RETVAL} -eq 0 ]; then
+			echo "started ${NAME} with pid $(cat ${PIDFILE})"
 		else
-			echo "starting ${NAME} failed. please see $LOGFILE "
+			echo "starting ${NAME} failed. please see ${LOGFILE}"
 		fi
 	fi
 }
 
 stop() {
-    if [ -f $PIDFILE ]; then
+    if [ -f ${PIDFILE} ]; then
```

Comment:
Again, you need to check that the PID is really alive, the fact the PID file is present is not enough.
Call status.